packages/cozy-client/src/models/dacc.js

```diff
@@ -71,3 +80,45 @@ export const sendMeasureToDACC = async (client, remoteDoctype, measure) => {
     throw error
   }
 }
+
+export const buildAggregateParams = params => {
+  const { measureName } = params
+  if (!measureName || typeof measureName !== 'string') {
+    throw new Error('Missing or wrong type parameter: measureName')
+  }
+  const startDate = params.startDate || new Date(0).toISOString()
+  const endDate = params.endDate || new Date(Date.now()).toISOString()
+  if (!isCorrectDateFormat(startDate) || !isCorrectDateFormat(endDate)) {
+    throw new Error('Date should be in YYYY-MM-DD format')
```

Comment:
Since this is used in a service, let's add all the needed informations to the log. It'll be easier to debug, no? 

```suggestion
log(
      'error',
      `Date should be in YYYY-MM-DD format but received: startDate: ${startDate} | endDate: ${endDate}`
    )
    throw new Error('Date should be in YYYY-MM-DD format')
```